docker-compose.yaml

```diff
@@ -28,12 +16,12 @@ services:
       args:
         OLLAMA_API_BASE_URL: '/ollama/api'
       dockerfile: Dockerfile
-    image: ollama-webui:latest
+    image: ghcr.io/ollama-webui/ollama-webui:main
```

Comment:
This compose file uses locally built container image (as per the lines above), I believe this shouldn't be replaced with ghcr.io image.

Reply:
This way users can build their own image tagged that way, obviously they cannot upload it on repository so it's not a problem.
Docker uses SHA fingerprint anyway to check the latest image so if user decide to do a --pull it will be forcibly replaced with the one from the repository.
I find more useful to have a choice without having to edit the docker-compose, if you want to build just pass the --build argument and compose will use the built image (because it's fingerprint will be different from the repo one), otherwise if you do not pass the --build it will use the repository image, much simpler and transparent to use imo.